libs/wire-api-federation/src/Wire/API/Federation/API/Galley.hs

```diff
@@ -38,7 +41,15 @@ import Wire.API.Federation.Util.Aeson (CustomEncoded (CustomEncoded))
 -- for the current list we need.
 
 data Api routes = Api
-  { getConversations ::
+  { -- | Create a new conversation
+    createConversation ::
+      routes
+        :- "federation"
+        :> Summary "Create a new conversation"
```

Comment:
This description is a bit misleading. This RPC does not really create a conversation, it just registers local users as part of one.

Reply:
I updated this. Thanks!